lib/oxidized/model/tplink.rb

```diff
@@ -18,13 +18,17 @@ class TPLink < Oxidized::Model
   end
 
   cmd :all do |cfg|
+    # remove unwanted paging line
+    cfg.gsub! /Press any key to contine.*/, ''
```

Comment:
Normally, we'd expect the pager handler at L8 to discard this after handling the pager. If the pager handler didn't match, then no key would've normally been pressed by the pager handler and the model should hang - whereas if it was, the prompt should've been removed. 

What situation leads to this being needed?

Reply:
I can't answer exactly. I experienced this behave for all the different tplink switches I own, and this line (sometimes followed by one or more special symbols) was part of the output, and the output did not hung at his point. (i.e. the config was always complete)
As you, I think the pager handler did something wrong, but I don't have the knowledge to fix this.
That was the reason for me just to remove the line in the output. I think this should be safe.

Reply:
However, I just realized that the current regex does not match all situations. So I will review and adapt.